Text-Grab/Views/FullscreenGrab.xaml.cs

```diff
@@ -495,8 +495,8 @@ private void RegionClickCanvas_MouseDown(object sender, MouseButtonEventArgs e)
         RegionClickCanvas.CaptureMouse();
         CursorClipper.ClipCursor(this);
         clickedPoint = e.GetPosition(this);
-        selectBorder.Height = 1;
-        selectBorder.Width = 1;
+        selectBorder.Height = 2;
+        selectBorder.Width = 2;
```

Comment:
because of this change line 599 will need to have a larger check size to see if the click was a single point. I tested using 5px instead of 3px for detecting of it was a small click and that worked.

previous: `bool isSmallClick = (regionScaled.Width < 3 || regionScaled.Height < 3);`

new: `bool isSmallClick = (regionScaled.Width < 5 || regionScaled.Height < 5);`

If you want to check it on your end we should be good.

Reply:
I modified this initial value because I found that if I hold down shift and drag from the beginning of the screenshot, an error will occur at line `435`, and the calculated value will be negative

```cs
clippingGeometry.Rect = new Rect(
  new System.Windows.Point(leftValue, topValue),
  new System.Windows.Size(selectBorder.Width - 2, selectBorder.Height - 2));
```

Reply:
Since the size of regionScaled is calculated by `selectBorder` through screen scaling, why not directly use the size of `selectBorder` to determine whether it is a `smallclick`? m.M11 and m.M22 I looked at it should be the screen scaling right, and the only thing that is obtained is the scaling of the main screen, and this is impossible to determine, may be 100% may be 200% may be 300%, if it is 300% that `smallclick` judgment is not satisfied which use regionScaled.Width < 5 || regionScaled.Height < 5 (of course, this situation is generally less), so either in the acquisition of the true scaling ratio calculated back to the original size, to determine whether it is the initial size, that in fact, it is equivalent to the size of the direct use of `regionScaled` to determine whether it is a small Wouldn't clicking be better?
> Of course this all assumes that I understand your `smallclick` as a left click and no drag, if that's not the definition of a small click on your side of the design, then it's fine!